.github/workflows/reusable_e2e_tests.yaml

```diff
@@ -18,10 +18,16 @@ on:
         type: boolean
         required: false
         default: true
-
-concurrency:
-  group: e2e-tests
-  cancel-in-progress: true
+      libsversion:
+        description: 'libs version to be tested, eg: master'
+        type: string
+        required: false
+        default: ${{ github.action_ref }}
```

Comment:
This means that if we call the workflow like `@master` it will default to `master`.